### DISCUSSION TITLE
how to get the 'dropdown' display correctly in table?

### DISCUSSION THREAD

**Discussion by walkthetalk** [ArthurSonzogni/FTXUI#787] [Q&A]
through constructing a Renderer like following code, we can get all components in table refresh automatically, but the dropdown list will result in the table expanded. how to make the dropdown list  display over the table?

Renderer(tbl_components, [&]() -> ftxui::Element { return contruct_table(tbl_components); })

**Comment by ArthurSonzogni** [comment]
You mean expanding the dropdown menu "outside" the component, without affecting its container.
There are no builtin support for this at the moment unfortunately.

**Reply by walkthetalk** [reply]
maybe we can implement it as a dialog?
